dev/integration_tests/web/web/index.html

```diff
@@ -8,19 +8,73 @@
   <meta content="IE=Edge" http-equiv="X-UA-Compatible">
 
   <title>Web Test</title>
+  <!-- iOS meta tags & icons -->
+  <meta name="apple-mobile-web-app-capable" content="yes">
+  <meta name="apple-mobile-web-app-status-bar-style" content="black">
+  <meta name="apple-mobile-web-app-title" content="Web Test">
   <link rel="manifest" href="manifest.json">
 </head>
 <body>
   <!-- This script installs service_worker.js to provide PWA functionality to
   application. For more information, see:
   https://developers.google.com/web/fundamentals/primers/service-workers -->
   <script>
+    var serviceWorkerVersion = null;
```

Comment:
Probably need to check for null since this won't be set during flutter run.

Reply:
Tested various `flutter run` modes. This seems to be working fine with `?v=null`. I'm open to stylistic improvements, but AFAICT it's not causing trouble.

Reply:
Why is the `serviceWorkerVersion` even required? Isn't the browser automatically checking the service worker file for byte differences?

Reply:
I could experiment with alternative approaches, but right now I'm using the version number to check that the currently active service worker is the one we want. The version is encoded in the `scriptURL` that I can read back from `reg.active`. If the versions don't match I force update. This happens when you pull down a new `index.html` but the worker is still from the old version of the app.